blobfuse/include/BlobStreamer.h

```diff
@@ -87,24 +87,18 @@ class StreamObject {
         }
 
         int IncRefCount() {
```

Comment:
Should we add a comment here to mention that users of this method must hold the lock before calling this? For this method, Dec and GetRefCount

Reply:
Will add comments